modules/nf-core/salsa2/main.nf

```diff
@@ -0,0 +1,38 @@
+process SALSA2 {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::salsa2=2.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/salsa2:2.3--py27hee3b9ab_0':
+        'quay.io/biocontainers/salsa2:2.3--py27hee3b9ab_0' }"
+
+    input:
+    // Input for salsa2 : bedfile from hicpro (nf-core/hic pipeline), draft assembly and draft assembly index
+    tuple val(meta), path(fasta), path(index)
+    path(bed)
+
+    output:
+    tuple val(meta), path("SALSA_output/scaffolds_FINAL.fasta"), emit: fasta
+    tuple val(meta), path("SALSA_output/scaffolds_FINAL.agp"), emit: agp
+//    path "versions.yml"           , emit: versions
```

Comment:
This must be emitted. If the tool doesn't print a version number itself, then one should be added to the code like
https://github.com/nf-core/modules/blob/master/modules/nf-core/homer/annotatepeaks/main.nf#L26, or if no release versions are available, then the commit hash if possible: https://github.com/nf-core/modules/blob/f703922050c6a74e68b35121304ac776271ac338/modules/nf-core/fastk/fastk/main.nf#L26

Reply:
Done 

https://github.com/scorreard/modules/blob/95321827d2be07508e8e26282ab35cf7efa06826/modules/nf-core/salsa2/main.nf#L33

Reply:
Done